test/Players.t.sol

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import { Test } from "../lib/forge-std/src/Test.sol";
+import { BeerLeagueBallers } from "../src/Players.sol";
+import "@openzeppelin-contracts/contracts/token/ERC1155/extensions/ERC1155Burnable.sol";
+
+contract MockERC1155 is ERC1155Burnable {
+    constructor() ERC1155("lol://lol") { }
+
+    function mint(address to, uint256 id, uint256 amount, bytes memory data) public virtual {
+        _mint(to, id, amount, data);
+    }
+
+    function mintBatch(address to, uint256[] memory ids, uint256[] memory amounts, bytes memory data) public virtual {
+        _mintBatch(to, ids, amounts, data);
+    }
+}
+
+contract PlayersTestBase is Test {
+    BeerLeagueBallers players;
+    MockERC1155 terminus;
+
+    uint256 adminPrivateKey = 0x1;
+    uint256 player1PrivateKey = 0x101;
+    uint256 player2PrivateKey = 0x102;
+    uint256 randomPersonPrivateKey = 0x201;
+
+    address admin = vm.addr(adminPrivateKey);
+    address player1 = vm.addr(player1PrivateKey);
+    address player2 = vm.addr(player2PrivateKey);
+    address randomPerson = vm.addr(randomPersonPrivateKey);
+
+    uint256 adminPoolID = 1;
+
+    function setUp() public virtual {
+        terminus = new MockERC1155();
+        terminus.mint(address(admin), adminPoolID, 1, "");
+
+        players = new BeerLeagueBallers(address(terminus), 1);
+    }
+}
+
+contract PlayersTestAdmin is PlayersTestBase {
+    function test_admin_can_add_profile_image() public {
+        uint256 initialImageCount = players.NumProfileImages();
+
+        vm.prank(admin);
+        players.addProfileImage("http://www.example.com");
+
+        assertEq(players.NumProfileImages(), initialImageCount + 1);
+    }
+
+    function test_non_admin_cannot_add_profile_image() public {
+        uint256 initialImageCount = players.NumProfileImages();
+
+        vm.prank(randomPerson);
+        vm.expectRevert("BeerLeagueBallers._enforceIsAdmin: not admin");
+        players.addProfileImage("http://www.example.com");
+
+        assertEq(players.NumProfileImages(), initialImageCount);
+    }
+
+    function test_admin_can_set_profile_image() public {
+        uint256 index = 1;
+        string memory initialImage = players.ProfileImages(index);
```

Comment:
Instead of using `https://example.com` here, I would append a suffix to the existing profile image. That leaves the test invariant to changes on the contract (e.g. for some reason we end up with example.com as the 0th profile image.

Reply:
If this is too pedantic, you can assert that the first one is not equal to `https://example.com`.